### PR TITLE
Fixed HTML5 Validator Errors

### PR DESCRIPTION
**Pull request by benmarten**
Hi Reza,

I have run the page through the Nu HTML Checker by W3C.org: https://validator.w3.org/nu/
A few Issus were found, that I have fixed. Now the project is 100% valid ;)
Feel free to merge these into the master branch ;)

Cheers,
Ben

### CODE REVIEW

File: layouts/account.jade [hunk 1]
```diff
@@ -41,7 +41,7 @@ html
           div.clearfix
 
     div.ajax-spinner
-      img(src='/media/ajax-pulse.gif')
+      img(src='/media/ajax-pulse.gif', alt='Loading ...')
```
Comment (jedireza) [inline layouts/account.jade]: Let's remove ` ...`

File: layouts/default.jade [hunk 2]
```diff
@@ -47,7 +47,7 @@ html
         div.clearfix
 
     div.ajax-spinner
-      img(src='/media/ajax-pulse.gif')
+      img(src='/media/ajax-pulse.gif', alt='Loading ...')
```
Comment (jedireza) [inline layouts/default.jade]: Let's remove ` ...`

File: views/index.jade [hunk 3]
```diff
@@ -9,11 +9,11 @@ block neck
 block body
   div.jumbotron
     h1 Success
-    p.lead
-      | Your Node.js website and user system is running. May the force be with you.
+    div.lead
+      p Your Node.js website and user system is running. May the force be with you.
```
Comment (jedireza) [inline views/index.jade]: Why is this change needed?

**Comment by jedireza** [comment]
Thanks so much! I left a few comments.

File: views/index.jade (hunk 3 above)
Comment (benmarten) [inline views/index.jade]: Since the p contains a div, it displays an error in the html validator. Indeed this SO answer clarifies, that it's not valid html: http://stackoverflow.com/a/10763952

File: layouts/default.jade (hunk 2 above)
Comment (benmarten) [inline layouts/default.jade]: I'm also not a big fan of it, but the alt tag is still considered required: "The  tag has two required attributes: src and alt." http://www.w3schools.com/tags/tag_img.asp

File: layouts/account.jade (hunk 1 above)
Comment (benmarten) [inline layouts/account.jade]: I'm also not a big fan of it, but the alt tag is still considered required: "The  tag has two required attributes: src and alt." http://www.w3schools.com/tags/tag_img.asp

Comment (jedireza) [inline layouts/account.jade]: Oh haha. I meant let's change `alt='Loading ...'` to `alt='Loading'`

File: views/index.jade (hunk 3 above)
Comment (jedireza) [inline views/index.jade]: Ah. Well the `p` didn't contain a `div`. That pipe character is a way jade let's us insert text while breaking to the next line. You can verify by inspecting the source of the live demo.

screen shot 2015-06-30 at 10 36 07 pm

Comment (benmarten) [inline views/index.jade]: Actually I think the inspector is wrong here, please view the plain source and you will see: 
screen shot 2015-07-01 at 12 50 29

Comment (jedireza) [inline views/index.jade]: My bad, yes the buttons are being put in that tag. Let's just pull those buttons back one indention instead.

```js
p.lead
  | Your Node.js website and user system is running. May the force be with you.
div
  a.btn.btn-primary.btn-lg(href='/signup/') Create an Account
  |  or&nbsp;
  | &nbsp;or&nbsp;
  a.btn.btn-warning.btn-lg(href='/login/forgot/') Reset Your Password
div.clearfix
```

**Comment by benmarten** [comment]
should be good now... please check

**Comment by jedireza** [comment]
:+1: Nice! Let's squash your commits and then we can merge it.

**Comment by benmarten** [comment]
Now we got it ;)

**Comment by jedireza** [comment]
Thank you so much!
